tools/gn

```diff
@@ -503,6 +506,10 @@ def parse_args(args):
                       help='Build the GLFW shell on supported platforms where it is not built by default.')
   parser.add_argument('--no-build-glfw-shell', dest='build_glfw_shell', action='store_const', const=False,
                       help='Do not build the GLFW shell on platforms where it is built by default.')
+  parser.add_argument('--build-embedder-examples', action='store_const', const=True,
+                      help='Build the example embedders using the Embedder API.')
+  parser.add_argument('--no-build-embedder-examples', dest='build_embedder_examples', action='store_const', const=False,
```

Comment:
Isn't the `no` version of a flag created automatically when it's a boolean?

Reply:
Good question. I just followed the example of the `--no-build-glfw-shell` flag above.

Reply:
Apparently, it isn't: `error: unrecognized arguments: --no-build-embedder-examples` when I remove it. Apparently, it is [possible in Python 3.9 according to the internet](https://stackoverflow.com/a/31347222). Leaving it as is.